### ISSUE TITLE
Layer normalization not done correctly

### ISSUE THREAD

**Issue by jalexvig** [ajarai/fast-weights#9]
In case of RNN-LN-FW model mean and standard deviation are incorrectly calculated as they would be for batch normalization:

```
mu = tf.reduce_mean(self.h_s, reduction_indices=0) # each sample
sigma = tf.sqrt(tf.reduce_mean(tf.square(self.h_s - mu), reduction_indices=0))
```

`self.h_s` has shape `[batch_size, 1, num_hidden]`

Whereas in the RNN-LN model they are calculated correctly as:

```
mean, var = tf.nn.moments(inputs, [1], keep_dims=True)
```

`inputs` has shape `[batch_size, num_hidden]`

**Comment by GokuMohandas** [comment]
nice catch Alex. Haven't worked with tensorflow in months, so hopefully my blind edit doesn't destroy everything lol.
